src/wp-admin/includes/ajax-actions.php

```diff
@@ -4588,6 +4588,32 @@ function wp_ajax_activate_plugin() {
 		wp_send_json_error( $status );
 	}
 
+	/**
+	 * Filters the data for the plugin's configuration workflow.
+	 *
+	 * @since 6.5.4
+	 *
+	 * @param array {
+	 *      An array of configuration data for the plugin.
+	 *
+	 *      @type string $url The URL to configure the plugin.
+	 *                        Must be an absolute URL beginning with the administration URL.
+	 *                        Default empty string for no configuration URL.
+	 * }
+	 */
+	$configuration_data = apply_filters( 'plugin_configuration_data_' . $status['slug'], array( 'url' => '' ) );
```

Comment:
```suggestion
	$configuration_data = apply_filters( "wp_plugin_configuration_data_{$status['slug']}", array( 'url' => '' ) );
```

* Prefixes the filter with WP because it's a WordPress filter
* CS fix per [coding standards for dynamic hooks](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#interpolation-for-naming-dynamic-hooks).

Reply:
Good catches! Thanks @peterwilsoncc. Suggestions done in https://github.com/WordPress/wordpress-develop/pull/6606/commits/d2749cd20858e7538fd27313d00cdb01d0f68e58.